ui/src/components/VaultSummary.jsx

```diff
@@ -61,9 +68,68 @@ const useStyles = makeStyles(theme => {
   };
 });
 
+const calcRatio = (priceRate, newLock, newBorrow) => {
+  const lockPrice = floorMultiplyBy(newLock, priceRate);
+  const newRatio = makeRatioFromAmounts(lockPrice, newBorrow);
```

Comment:
This discards precision unnecessarily. How about
```suggestion
  const newRatio = makeRatioFromAmounts(
    multiply(newLock, priceRate.numerator),
    multiply(newBorrow, priceRate.denominator),
  );
```

If the old value can be used in calculating the new value, it would make sense to use the new `quantize` function in `ratio.js` to reduce the size of the numbers, but that's best done after all the multiplication is done, and not part-way through.